packages/orbit-components/src/HorizontalScroll/components/ArrowButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import cx from "clsx";
+
+type Props = React.PropsWithChildren<{
+  className: string;
+  isHidden: boolean;
+  onClick: () => void;
+}>;
+
+const ArrowButton = ({ children, className, isHidden, onClick }: Props) => {
+  return (
+    <button
+      className={cx(
+        "z-default absolute flex h-full items-center",
```

Comment:
I changed this to `z-default` intentionally since arrows are present after the elevation shadows in the markup so they’ll get rendered above them even though having the same `z-index`. In general I think having the z-index scale as flat as possible is better for maintenance.

Reply:
Alright, agreed! Thank you!